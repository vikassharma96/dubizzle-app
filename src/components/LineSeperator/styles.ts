import {StyleSheet} from 'react-native';
import colors from '../../styles/colors';

const styles = StyleSheet.create({
  divider: {
    height: 1,
    marginVertical: '2.5%',
    backgroundColor: colors.lightGray,
  },
});

export default styles;
