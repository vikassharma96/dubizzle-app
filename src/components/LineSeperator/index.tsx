import React from 'react';
import {View, ViewStyle} from 'react-native';
import styles from './styles';

type IProps = {
  style?: ViewStyle;
};

const LineSeperator = (props: IProps) => {
  return <View style={[styles.divider, {...props.style}]} />;
};

export default LineSeperator;
