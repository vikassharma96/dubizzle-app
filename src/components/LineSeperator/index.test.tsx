import React from 'react';
import renderer from 'react-test-renderer';
import LineSeperator from '.';

describe('@components/LineSeperator', () => {
  const component = <LineSeperator />;

  it('should match snapshots', () => {
    const componentTree = renderer.create(component).toJSON();
    expect(componentTree).toMatchSnapshot();
  });
});
