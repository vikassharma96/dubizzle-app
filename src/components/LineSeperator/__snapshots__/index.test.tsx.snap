// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@components/LineSeperator should match snapshots 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#ccc",
        "height": 1,
        "marginVertical": "2.5%",
      },
      {},
    ]
  }
/>
`;
